src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1333,6 +1369,10 @@ void OrbitMainWindow::on_actionCaptureOptions_triggered() {
                     dialog.GetLimitLocalMarkerDepthPerCommandBuffer());
   settings.setValue(kMaxLocalMarkerDepthPerCommandBufferSettingsKey,
                     QString::number(dialog.GetMaxLocalMarkerDepthPerCommandBuffer()));
+  settings.setValue(kTracepointCallstackCollectionMethod,
+                    QString::number(dialog.GetTracepointCallstackMethod()));
+  settings.setValue(kIsPureTracepoint,
+                    QString::number(static_cast<int>(dialog.GetPureOrNotTracepoint())));
```

Comment:
Why `QString::number`? Just passing in ints should work. It does not?

Reply:
Done.